### ISSUE TITLE
Shelly 1 Plus Mini Gen 3

### ISSUE THREAD

**Issue by Maigus2510** [alexryd/homebridge-shelly-ng#120]
Hello,

is this Device supported:

https://shellyparts.de/products/shelly-mini-plus-1pm

i kann not link ist to the HomeKit with this Script. The Plugin show this Failures:

22/03/2024, 15:56:45] [Shelly NG] Initializing ShellyNG platform...
[22/03/2024, 15:56:45] [Shelly NG] mDNS device discovery started
[22/03/2024, 15:56:45] [Shelly NG] [shelly1pmminig3-54320454a2e8] Unknown device of model "S3SW-001P8EU" discovered.
[22/03/2024, 16:01:42] [Shelly NG] Failed to add discovered device (id: shelly1pmminig3-ecda3bc0778c): shutting down in 2877 ms
[22/03/2024, 16:01:49] [Shelly NG] [shelly1pmminig3-ecda3bc0778c] Unknown device of model "S3SW-001P8EU" discovered.
[22/03/2024, 16:16:29] [Shelly NG] Initializing ShellyNG platform...
[22/03/2024, 16:16:30] [Shelly NG] mDNS device discovery started
[22/03/2024, 16:16:30] [Shelly NG] Failed to add discovered device (id: shelly1pmminig3-54320454a2e8): Unexpected device ID (returned: shelly1pmminig3-ecda3bc0778c, expected: shelly1pmminig3-54320454a2e8)
[22/03/2024, 17:20:16] [Shelly NG] [shelly1pmminig3-ecda3bc0778c] Unknown device of model "S3SW-001P8EU" discovered.

Everyone can help me with this?

Thank you :-)

**Comment by cwiemann** [comment]
Agreed. Would be great to get this device and other Gen3 devices added to the compatibility list!

**Comment by luca-vogels** [comment]
Having the same issue. Cannot use my Shelly 1PM Mini Gen3.
Log says "Unknown device of model S3SW-001P8EU discovered"

**Comment by apartmedia** [comment]
Did you try the forked repo homebridge-shelly-ds9?
This should support Gen3 devices.
alexryd is afk so this repo seems to be dead… (kind of)
